versioned_docs/version-v1.4/integrations/integrations.md

```diff
@@ -0,0 +1,25 @@
+---
```

Comment:
please add this doc into Next version beside version-v1.4.

Reply:
@rambohe-ch This doc has been added into Next version.